pylint/checkers/imports.py

```diff
@@ -850,6 +866,83 @@ def _check_imports_order(
                         )
         return std_imports, external_imports, local_imports
 
+    def _get_out_of_order_string(
+        self,
+        third_party_imports: list[tuple[ImportNode, str]] | None,
+        first_party_imports: list[tuple[ImportNode, str]] | None,
+        local_imports: list[tuple[ImportNode, str]] | None,
+    ) -> str:
+        # construct the string listing out of order imports used in the message
+        # for wrong-import-order
+        third_party_str = (
+            "third party import(s) "
+            + ", ".join(
+                [
+                    '"' + self._get_full_import_name(tpi) + '"'
+                    for tpi in third_party_imports
+                ]
+            )
+            if third_party_imports
+            else ""
+        )
+        first_party_str = (
+            "first party import(s) "
```

Comment:
```suggestion
            "first party import{plural} "
```

Reply:
Suggested changes using `plural` to make message cleaner have been resolved in commit 2333af9